Model/Ui/ConfigProvider.php

```diff
@@ -97,7 +97,7 @@ public function getConfig()
                         $plan['key'] = $planConfig->planKey();
                         $plan['logo'] = $planConfig->logoFileName();
                         $plan['installments'] = $pe->getEligibility()->getPaymentPlan();
```

Comment:
here, it seems to be more than just `installments` because API `$eligibility->payment_plan` is stored here (as `array()`)

could you change the name of this key to match what is stored here please ?